### ISSUE TITLE
Support for using personal access token

### ISSUE THREAD

**Issue by mink-ang** [teameh/bitrise-step-bitbucket-server-post-build-status#3]
Instead of username and password, bitbucket server rest api also supports the use of personal access token. Just need to include them in the authorization header. I can contribute and create a PR if you'd like

**Comment by teameh** [comment]
Yeah that would be great. I've got no bitbucket server setup that supports the token so I could not test this.

**Comment by teameh** [comment]
@mink-ang any luck with this? Let me know if/how I can help.

**Comment by teameh** [comment]
Anyone else who would like to implement this?
